Lib/unittest/mock.py

```diff
@@ -2754,7 +2754,19 @@ def create_autospec(spec, spec_set=False, instance=False, _parent=None,
         raise InvalidSpecError(f'Cannot autospec a Mock object. '
                                f'[object={spec!r}]')
     is_async_func = _is_async_func(spec)
-    _kwargs = {'spec': spec}
+
+    placeholder = object()
```

Comment:
No need to instantiate this each time. I think we have `mock.Sentinel` already and likely have one of these further up in the module, please have a dig.